src/main/java/technology/tabula/Page.java

```diff
@@ -116,19 +116,22 @@ public Page getArea(float top, float left, float bottom, float right) {
         return this.spatial_index.contains(area);
     }
     
-    public List<TextElement> getText(float top, float left, float bottom, float right) {
+    /** @deprecated use {@linkplain #getText(Rectangle)} instead  */
+    @Deprecated public List<TextElement> getText(float top, float left, float bottom, float right) {
         return this.getText(new Rectangle(top, left, right - left, bottom - top));
     }
 
-    public Integer getRotation() {
+    /** @deprecated with no replacement  */
+    @Deprecated public Integer getRotation() {
```

Comment:
This is used in the App: https://github.com/tabulapdf/tabula/blob/master/lib/tabula_java_wrapper.rb#L102